tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,538 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "```yaml\n",
+    "services:\n",
+    "  ...\n",
+    "  elasticsearch:\n",
+    "    image: \"docker.elastic.co/elasticsearch/elasticsearch:7.9.2\"\n",
+    "    ports:\n",
+    "      - 9200:9200\n",
+    "    restart: on-failure\n",
+    "    environment:\n",
+    "      - discovery.type=single-node\n",
+    "      - \"ES_JAVA_OPTS=-Xms1024m -Xmx1024m\"\n",
+    "    healthcheck:\n",
+    "        test: curl --fail http://localhost:9200/_cat/health || exit 1\n",
+    "        interval: 10s\n",
+    "        timeout: 1s\n",
+    "        retries: 10\n",
+    "  ...\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "5. Launch Elasticsearch.\n",
+    "\n",
+    "Go to the directory where `docker-compose.yml` is and start the Elasticsearch container. You might have realised that there are settings for haystack and ui containers provided in the `docker-compose.yml`. They are not necessary for in this tutorial. Run `docker-compose up` with the `elasticsearch` argument so that only the Elasticsearch container launches."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "cd haystack\n",
+    "docker-compose up elasticsearch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "6. Check Elasticsearch health.\n",
+    "\n",
+    "Launching Elasticsearch takes some time, so, make sure you have a `healthy` Elasticsearch container before continuing. You can check the container health with the `docker ps` command. A `healthy` container should have a healthy status on port 9200.\n",
+    "\n",
+    "![command line output of `docker ps`](https://github.com/deepset-ai/haystack-tutorials/blob/main/tutorials/img/tutorial20_elasticsearch_healthy.png?raw=true)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "docker ps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Pipeline YAML File\n",
+    "\n",
+    "YAML files are widely used for configurations. Haystack enables defining pipelines as YAML files and the `load_from_yaml()` method loads pipelines from YAML file. In a YAML file, the `components` section defines all pipeline nodes and `pipelines` section defines how these nodes are connected to each other to form a pipeline. Let's start with defining query and indexing pipelines."
```

Comment:
```suggestion
    "YAML files are widely used for configurations. You can define Haystack pipelines as YAML files and then use the `load_from_yaml()` method to load pipelines from the YAML file. In a YAML file, the `components` section defines all pipeline nodes and the `pipelines` section defines how these nodes are connected to each other to form a pipeline. Let's start with defining query and indexing pipelines."
```

It's recommended not to say your product "enables" people to do something. Instead, you can say they can do something with the product.

Reply:
Hi @agnieszka-m, here, the user doesn't use the `load_from_yaml()` method, it is used by Haystack. I wanted to explain the process a little bit so that they can see the connection between a YAML file and python objects we use in Haystack. What do you think about this 👇 ?

```suggested
YAML files are widely used for configurations. You can define Haystack pipelines as YAML files and then `load_from_yaml()` method loads the pipeline defined in YAML into a Python object. In a YAML file, the `components` section defines all pipeline nodes and the `pipelines` section defines how these nodes are connected to each other to form a pipeline. Let's start with defining query and indexing pipelines.
```